core/store/src/db.rs

```diff
@@ -46,6 +46,9 @@ impl Into<io::Error> for DBError {
     }
 }
 
+/// Number of DBCol columns.
+static DB_NUM_COLS: u64 = 50;
```

Comment:
@matklad This can probably be written in a better way.

Reply:
wait I think `NUM_COLS` already exists in this file

Reply:
@bowenwang1996 It is, but it was in a wrong place.